src/core/engines/Cesium/Feature/Raster/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { ImageryLayerCollection, WebMapServiceImageryProvider } from "cesium";
+import { useEffect, useMemo } from "react";
+import { useCesium } from "resium";
+
+import type { RasterAppearance } from "../../..";
+import { extractSimpleLayerData, type FeatureComponentConfig, type FeatureProps } from "../utils";
+
+export type Props = FeatureProps<Property>;
+export type Property = RasterAppearance;
+
+const imageryProviders = {
+  wms: WebMapServiceImageryProvider,
+};
+
+export default function Raster({ isVisible, layer, property }: Props) {
+  const { layers, zIndex, minimumLevel, maximumLevel, credit } = property ?? {};
+  const { viewer } = useCesium();
+  const [type, url] = useMemo((): [RasterAppearance["type"], string | undefined] => {
+    const data = extractSimpleLayerData(layer);
+    const type = layer?.raster?.type;
+    const url = layer?.raster?.url;
+    return [type ?? (data?.type as RasterAppearance["type"]), url ?? data?.url];
+  }, [layer]);
+
+  const imageryProvider = type ? imageryProviders[type] : undefined;
```

Comment:
```suggestion
  const imageryProvider = type ? ImageryProviders[type] : undefined;
```

Reply:
I fixed it
[chore: rename](https://github.com/reearth/reearth-web/pull/387/commits/41c7093f58cd509e1e8b401340bf4c41504d176a)